spec/controllers/criteria_controller_spec.rb

```diff
@@ -1031,4 +1031,52 @@
       let(:params) { { course_id: course.id, assignment_id: assignment.id } }
     end
   end
+
+  context '#update_position' do
+    context 'when there exists criteria not under current assignment' do
+      let(:assignment2) { create :assignment }
+      let(:rubric_criterion) do
+        create(:rubric_criterion,
+               assignment: assignment,
+               position: 1,
```

Comment:
You don't need to specify a name for the criteria, as these will be automatically generated by the factory. In general it's good to omit specifying attributes that don't actually matter to the test case, since it keeps the test code shorter. (In fact, if you remove this attribute, each `let` statement should fit onto a single line!)